File: src/runners/Runner.ts
import { findMaxConsecutiveOnes } from "../answers/typescript/easy/MaxConsecutiveOnes";

class Runner {
    public run(): void {
        console.log("============ Welcome to TYPESCRIPT section ============");
        console.log("Max consecutive ones:", findMaxConsecutiveOnes([1, 1, 0, 1, 1, 1]));
    }
}

// Entry execution (like main method)
new Runner().run();

// run: npx ts-node src\runners/Runner.ts

File: src/runners/Runner.java
package runners;

import answers.java.easy.*;

public class Runner {
    public static void main(String[] args) {
        System.out.println("============ Welcome to JAVA section ============");
        System.out.println("Max consecutive ones:" + new MaxConsecutiveOnes().findMaxConsecutiveOnes(new int[] { 1, 1, 0, 1, 1, 1 }));
    }
}

// compile: javac src\runners\Runner.java
// run: java src\runners\Runner.java
